src/main/kotlin/com/devfloor/untitled/hashtag/domain/HashtagRepository.kt

```diff
@@ -2,4 +2,6 @@ package com.devfloor.untitled.hashtag.domain
 
 import org.springframework.data.jpa.repository.JpaRepository
 
-interface HashtagRepository : JpaRepository<Hashtag, Long>
+interface HashtagRepository : JpaRepository<Hashtag, Long>{
+    fun findByNameIn(hashtagNameList: List <String>): List<Hashtag>
```

Comment:
1. 이 부분이 엊그제인가 슬랙에 공유해주셨던 query를 사용하는?? 부분이 맞나요?
현재는 쿼리가 필요 없기 때문에 `In` 은 없애주면 어떨까요?

2. 이 부분 또한 name에 해당하는 것들 전부 반환하기 때문에 findAllByName이 적절할 것 같아요!

3. 변수명에 List와 같이 특정 타입을 유추할수 있는 용어를 지양하는 것이 좋다고 합니다. ! 타입이 추후 변경되면 메서드명까지 수정해야하기 때문에요. 따라서 매개변수부분 hashtagNames로 변경하는 것도 좋을거같습니다
```suggestion
    fun findAllByName(hashtagNames: List <String>): List<Hashtag>
```

Reply:
이 부분은 주원이형 피드백에 이어서 달게요!!

이건 뒤에 In이 들어가는게 맞을 것 같아요!! 일단 IDE에서 자동완성 해주는 네이밍이 이처럼 생성될겁니다(아마도?)
그리고 실제로 쿼리도 In절로 날라갈거에요.
위에서 제안해 준 findAllByName이라는 메서드로 생성하면
```kotlin
fun findAllByName(name: String) : List<Hashtag>
```

이런 식일겁니다. 즉 하나의 Name에 해당하는 모든 Hashtag가 조회될 거에요! (물론 비즈니스적으로는 불가능하지만 JPA에서는 이렇게 인식할겁니다)

```kotlin
fun findAllByNameIn(names: List<String>): List<Hashtag>
```

이건 뒤에 names가 in절 조건으로 들어가서 조회될 것이고요.
그래서 위 상황에서는 마지막에 In을 붙이는게 맞을 것 같네요!!

추가적으로 3번에 List라는 이런 특정 타입을 지칭하는 용어를 사용해서 명명하지 않는게 좋다는 부분은 주원이형이 잘 설명해준 것 같아요!!
단순하게 List에서 Set으로 자료구조를 바꾼다면, 기존에 이름이 names가 아닌 nameList라면 명명까지 바꿔야겠죠? 그런데 names였다면 복수형이기 때문에 굳이 명명을 바꿀 필요는 없었을겁니다!! 그래서 주로 불용어를 명명에 사용하지 않습니다!!!

Reply:
제가 In이라는 것에 대해 잘못 이해하고 있었네요 ! 준형님이 올려주신 자료 다시한번 보고 이해했습니다!